src/staff-administrator/components/staff-user-details/staff-user-details.component.ts

```diff
@@ -79,7 +79,8 @@ export class StaffUserDetailsComponent {
       ...this.userDetails,
       first_name: '',
       last_name: '',
-      email_id: ''
+      email_id: '',
+      suspended: false,
```

Comment:
Copying always moves user who is suspended as false? Is this the requirement?

Reply:
Not sure if it's a requirement. However, it seems that you cannot add an user with suspended: true.

```
{
    "errorCode": 400,
    "status": "Bad Request",
    "errorMessage": "3 : There is a problem with your request. Please check and try again",
    "errorDescription": "There is no user present to suspend. Please try again or check with HMCTS Support Team",
    "timeStamp": "25-04-2023 12:39:58.591"
}

```